src/test/rustdoc-gui/font-weight.goml

```diff
@@ -13,6 +13,8 @@ goto: file://|DOC_PATH|/test_docs/struct.Foo.html
 assert-css: (".impl-items .method", {"font-weight": "600"}, ALL)
 
 goto: file://|DOC_PATH|/lib2/trait.Trait.html
+assert-count: (".type-decl > pre.trait > code > a.constant", 1)
+assert-css: (".type-decl > pre.trait > code > a.constant", {"font-weight": "400"})
```

Comment:
I'm definitely not sure that it'll it because it's the inner content which had the `font-weight` change, not the container.

Reply:
Good point, I have now changed it to use XPath with `/text()`.